tgui/babel.config.js

```diff
@@ -23,7 +23,7 @@ const createBabelConfig = (options) => {
       ...presets,
     ].filter(Boolean),
     plugins: [
-      [require.resolve('@babel/plugin-proposal-class-properties'), {
+      [require.resolve('@babel/plugin-transform-class-properties'), {
```

Comment:
proposal is deprecated

> This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.

https://www.npmjs.com/package/@babel/plugin-proposal-class-properties